emails/models.py

```diff
@@ -132,6 +132,7 @@ class Profile(models.Model):
     # Empty string means the profile was created through relying party flow
     created_by = models.CharField(blank=True, null=True, max_length=63)
     sent_welcome_email = models.BooleanField(default=False)
+    last_engagement = models.DateTimeField(blank=True, null=True, db_index=True)
```

Comment:
Why index this column?

Reply:
I'm almost sure we'll query this column for number of Daily/Weekly/Monthly active users. So I expect SQL like: `SELECT count(id) FROM emails_profile WHERE last_engagement >= now() - INTERVAL "1 day"`, etc.